projectName-web/src/main/resources/META-INF/spring/spring-mvc.xml

```diff
@@ -36,14 +36,12 @@
         <mvc:interceptor>
             <mvc:mapping path="/**" />
             <mvc:exclude-mapping path="/resources/**" />
-            <mvc:exclude-mapping path="/**/*.html" />
             <bean
                 class="org.terasoluna.gfw.web.logging.TraceLoggingInterceptor" />
         </mvc:interceptor>
         <mvc:interceptor>
             <mvc:mapping path="/**" />
             <mvc:exclude-mapping path="/resources/**" />
-            <mvc:exclude-mapping path="/**/*.html" />
             <bean
                 class="org.terasoluna.gfw.web.token.transaction.TransactionTokenInterceptor" />
         </mvc:interceptor>
```

Comment:
- [ ] Same as https://github.com/terasolunaorg/terasoluna-gfw-web-blank/pull/363#discussion_r181329442

Reply:
@yoshikawaa
I fixed it because I have the same opinon as your comment. 
a6bf294e75aef6229f52c48e24f6ffe05ebdc0a6

Reply:
@yoshikawaa 
I confirmed the fixing.